cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/adls/ADLSStorageClient.java

```diff
@@ -0,0 +1,110 @@
+package com.linkedin.openhouse.cluster.storage.abs;
+
+import com.azure.storage.file.datalake.DataLakeFileClient;
+import com.linkedin.openhouse.cluster.storage.StorageClient;
+import com.linkedin.openhouse.cluster.storage.StorageType;
+import com.linkedin.openhouse.cluster.storage.configs.StorageProperties;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.PostConstruct;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.iceberg.azure.adlsv2.ADLSFileIO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+/**
```

Comment:
The comment does not match the implementation below.
The implementation is using DataLakeFileClient but the comments mentions BlobClient,